doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -472,77 +291,180 @@ You can schedule actors/tasks on the placement group using
+  However, when the actor is created, it will ignore the placement group.
 
-  .. tabbed:: Python
+The actor is scheduled now! One bundle can be used by multiple tasks and actors (i.e., bundle to task/actor is one to many relationships). 
+In this case, since the actor uses 1 CPU, there's a remaining 1 GPU from the bundle. 
+You can verify this from the CLI command ``ray status``. You can see the 1 CPU is reserved by the placement group, and 1.0 is used (by the actor we created).
 
-      .. literalinclude:: ../doc_code/placement_group_capture_child_tasks_example.py
-        :language: python
+.. code-block:: bash
+
+  ray status
+
+.. code-block:: bash
 
-  .. tabbed:: Java
+  Resources
+  ---------------------------------------------------------------
+  Usage:
+  1.0/2.0 CPU (1.0 used of 1.0 reserved in placement groups) <---
+  0.0/2.0 GPU (0.0 used of 1.0 reserved in placement groups)
+  0B/4.29GiB memory
+  0B/2.00GiB object_store_memory
 
-      It's not implemented for Java APIs yet.
+  Demands:
+  (no resource demands)
 
-  When ``placement_group_capture_child_tasks`` is True, and if you'd like to avoid scheduling
-  child tasks/actors, you should specify the below option when you call child tasks/actors.
+You can also verify the actor is created using ``ray list actors``
 
-  .. code-block:: python
+.. code-block:: bash
 
-    @ray.remote
-    def parent():
-        # In this case, the child task won't be
-        # scheduled with the parent's placement group.
-        ray.get(child.options(
-            scheduling_strategy=PlacementGroupSchedulingStrategy(
-                placement_group=None)).remote())
+  # This API is only available when you download Ray via `pip install "ray[default]"`
+  ray list actors --detail
 
-You can remove a placement group at any time to free its allocated resources.
+.. code-block:: bash
+
+  -   actor_id: b5c990f135a7b32bfbb05e1701000000
+      class_name: Actor
+      death_cause: null
+      is_detached: false
+      job_id: '01000000'
+      name: ''
+      node_id: b552ca3009081c9de857a31e529d248ba051a4d3aeece7135dde8427
+      pid: 8795
+      placement_group_id: d2e660ac256db230dbe516127c4a01000000 <------
+      ray_namespace: e5b19111-306c-4cd8-9e4f-4b13d42dff86
+      repr_name: ''
+      required_resources:
+          CPU_group_d2e660ac256db230dbe516127c4a01000000: 1.0
+      serialized_runtime_env: '{}'
+      state: ALIVE
+
+Since we have a remaining 1 GPU, let's create a new actor that requires 1 GPU.
+This time, we will also specify the ``placement_group_bundle_index``. Each bundle is given an "index" within the placement group.
+E.g., A placement group of 2 bundles ``[{"CPU": 1}, {"GPU": 1}]`` will have index 0 bundle ``{"CPU": 1}`` 
+and index 1 bundle ``{"GPU": 1}``. Since we only have 1 bundle, we will only have index 0. If you don't specify a bundle, the actor/task
```

Comment:
```suggestion
and index 1 bundle ``{"GPU": 1}``. Since we only have 1 bundle, we only have index 0. If you don't specify a bundle, the actor (or task)
```